rest/taskana-rest-spring/src/test/java/pro/taskana/common/rest/AccessIdControllerIntTest.java

```diff
@@ -61,6 +61,17 @@ void testQueryGroupsByCn() {
         .containsExactly("cn=ksc-users,cn=groups,OU=Test,O=TASKANA");
   }
 
+  @Test
+  void should_returnEmptyResults_ifInvalidCharacterIsUsedInCondition() {
```

Comment:
Sorry :sweat_smile: : I don't understand the test name. 
What in `ksc-teamleads,cn=groups` is invalid?

Reply:
It is supposed to be a cn and nothing more. LDAP injection aims to prevent, that you can change the search path implictly, as I try in this case.

Reply:
thanks! :)